### PR TITLE
Fix loading woff 1 fonts

### PR DESCRIPTION
**Pull request by brianpopow**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/Fonts/pulls) open
- [x] I have verified that I am following matches the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description

This fixes an issue loading woff1 files: in some cases, its necessary to seek back in the stream to load strings. For that the compressed stream is read completely.

### CODE REVIEW

File: tests/SixLabors.Fonts.Tests/GlyphTests.cs [hunk 1]
```diff
@@ -126,9 +126,32 @@ public void RenderColrGlyphTextRenderer()
...
+            Assert.True(rendererTtf.ControlPoints.SequenceEqual(rendererWoff.ControlPoints));
+        }
+
 #if NETCOREAPP3_0_OR_GREATER
         [Fact]
-        public void RenderWoff2Glyphs_EqualToTtfGlyphs()
+        public void RenderWoff2Glyphs_IsEqualToTtfGlyphs()
         {
             Font fontTtf = new FontCollection().Add(TestFonts.OpenSansFile).CreateFont(12);
             Font fontWoff2 = new FontCollection().Add(TestFonts.OpenSansFileWoff2).CreateFont(12);
```
Comment (JimBobSquarePants) [inline tests/SixLabors.Fonts.Tests/GlyphTests.cs:157]: I was wondering if you'd changed this as the new font will fail to load.

**Review by JimBobSquarePants — APPROVED** [review]
You're quick! Nice job 👍

File: tests/SixLabors.Fonts.Tests/GlyphTests.cs (hunk 1 above)
Comment (brianpopow) [reply tests/SixLabors.Fonts.Tests/GlyphTests.cs:157]: I wanted to change this to the new files once we figured out the issue with the woff2 file

Comment (JimBobSquarePants) [reply tests/SixLabors.Fonts.Tests/GlyphTests.cs:157]: Looking at that now will push a branch shortly.

**Pull request merged**